<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PidSelectDialog</class>
 <widget class="QDialog" name="PidSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>289</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>441</width>
    <height>289</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>441</width>
    <height>289</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Selección de PIDs</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>254</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_sensor_O2">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>240</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selecciona las unidades en las que se desea loggear los sensores de oxígeno.&lt;/p&gt;&lt;p&gt;Ratio: Sin unidades, equivalente a Lambda.&lt;/p&gt;&lt;p&gt;Voltios y/o Amperios: En Voltios o miliamperios según el caso.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="title">
    <string>Sensor O2</string>
   </property>
   <widget class="QRadioButton" name="radioButton_ratio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ratio</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_voltage">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>15</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltios y/o Amperios</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_select">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>240</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Deseleccionar
todos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PidSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PidSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
